### PR TITLE
Auctions allocate fees instead of withdraw reserve

### PR DESCRIPTION
**Pull request by trmid**

### CODE REVIEW

File: test/PrizePool.t.sol [hunk 1]
```diff
@@ -65,37 +65,37 @@ contract PrizePoolTest is Test {
...
   /// @param amount The amount of tokens contributed
   event ContributePrizeTokens(address indexed vault, uint24 indexed drawId, uint256 amount);
 
-  /// @notice Emitted when an address withdraws their claim rewards
+  /// @notice Emitted when an address withdraws their prize claim rewards.
   /// @param to The address the rewards are sent to
   /// @param amount The amount withdrawn
   /// @param available The total amount that was available to withdraw before the transfer
-  event WithdrawClaimRewards(address indexed to, uint256 amount, uint256 available);
+  event WithdrawRewards(address indexed to, uint256 amount, uint256 available);
```
Comment (PierrickGT) [inline test/PrizePool.t.sol:88]: We can also log `msg.sender` here.

File: test/PrizePool.t.sol [hunk 2]
```diff
@@ -974,6 +981,47 @@ contract PrizePoolTest is Test {
...
+    assertApproxEqAbs(prizeAmount, (10e18 * TIER_SHARES) / (4 * prizePool.getTotalShares()), 100);
+
+    vm.expectEmit();
+    emit ClaimedPrize(address(this), winner, recipient, 1, 1, 0, uint152(prize), fee, address(this));
+    assertEq(prizePool.claimPrize(winner, 1, 0, recipient, fee, address(this)), prizeAmount);
+    assertEq(prizeToken.balanceOf(recipient), prize, "recipient balance is good");
+    assertEq(prizePool.claimCount(), 1);
+
+    // Check if claim fees are accounted for
+    // (if they aren't anyone can call contributePrizes with the unaccounted fee amount and basically take it as their own)
```
Comment (PierrickGT) [inline test/PrizePool.t.sol:1004]: I think you meant `contributePrizeTokens`.
```suggestion
    // (if they aren't anyone can call contributePrizeTokens with the unaccounted fee amount and basically take it as their own)
```

**Review by PierrickGT — CHANGES_REQUESTED** [review]

File: test/PrizePool.t.sol (hunk 1 above)
Comment (trmid) [reply test/PrizePool.t.sol]: resolved with: [17ad8cc](https://github.com/GenerationSoftware/pt-v5-prize-pool/pull/69/commits/17ad8cc2f00a67133ea3c260eaa566ea41c328ff)

File: test/PrizePool.t.sol (hunk 2 above)
Comment (trmid) [reply test/PrizePool.t.sol]: resolved with: [17ad8cc](https://github.com/GenerationSoftware/pt-v5-prize-pool/pull/69/commits/17ad8cc2f00a67133ea3c260eaa566ea41c328ff)

**Review by PierrickGT — APPROVED** [review]

**Pull request merged**